tests/basic_test.go

```diff
@@ -0,0 +1,71 @@
+/*
+ * This file is part of the nmpolicy project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2021 Red Hat, Inc.
+ *
+ */
+
+package tests
+
+import (
+	"testing"
+
+	assert "github.com/stretchr/testify/require"
```

Comment:
Why do you use `testify` and not `ginkgo\gomega` like we do for all our other projects?

Reply:
We don't need ginkgo/gomega just an assert library and son subtests is enough.

Reply:
In kubernetes-nmstate we do use ginko/gomega. What is the reason it is needed there but not here?

Reply:
IMO it was done just because k8s and kubevirt/kubevirt used them.

But I suggest asking the question from the other direction.
Why do we need to use ginkgo & gomega in this project? I personally see no reason, but if there are points you think are now missing, I will be glad to hear.

Reply:
the golnag "subtest" did not exist when kubernetes started and ginkgo/gomega appear to overcome stuff like that, now that we bave subtest where we can express the test name with a string we don't need this big ginkgo asbtraction to use a string to put names.

Also show table testing is implemented at ginkgo is not good, forcing you to to put the implementation before the tests does not make much sense, also at ginkg when you go with DescribeTable you don't have the BeaforeEach/AfterEach anymore.

So now that golang has the subtest feature we don't need ginkgo/gomega, testify has also some kind of setup/teardown feature and suites too we waaaay less abstraction.